Objects/methodobject.c

```diff
@@ -55,17 +66,48 @@ PyCFunction_NewEx(PyMethodDef *ml, PyObject *self, PyObject *module)
         case METH_O:
             vectorcall = cfunction_vectorcall_O;
             break;
+        case METH_FASTCALL | METH_KEYWORDS | METH_METHOD:
```

Comment:
nitpick: the documentation uses the flags in a different order, I prefer this order:

```suggestion
        case METH_METHOD | METH_FASTCALL | METH_KEYWORDS:
```